sarathi/core/datatypes/block.py

```diff
@@ -1,9 +1,12 @@
 """Token blocks."""
 
-from typing import List
+from typing import List, Tuple, Union
+from torch import Tensor
+
 
 _BLANK_TOKEN_ID = -1
 
+KVCache = Union[Tuple[Tensor, Tensor], Tensor]
```

Comment:
nit: let's make sure we are don't redefine this at multiple places

Reply:
Done, removed from the other places